client/js/store/modules/auth.js

```diff
@@ -1,8 +1,10 @@
 import { io } from '../../plugins/sails.js';
+import { ROUTE_NAME_LOBBY, ROUTE_NAME_GAME } from '@/router';
 
 export default {
   state: {
-    authenticated: false,
+    // This value will ONLY be null on the initial load
+    authenticated: null,
```

Comment:
This is really important. We basically need to track 3 values here:
- `null` no auth attempt. This is a meaningful choice and purposely set by us so we can differentiate the initial load from a user being authed or not. This allows us to do auth logic once on the client side and avoid making the same request (or nooping) every route.
- `false` user is not authed (`clearAuth` also sets this)
- `true` user is authed (`authSuccess` also sets this)